index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express")
+const { GracefulShutdownServer } = require("medusa-core-utils")
+
+const loaders = require("@medusajs/medusa/dist/loaders/index").default
+
+;(async() => {
+  async function start() {
+    const app = express()
+		const directory = process.cwd()
+
+    try {
+      const { dbConnection, container } = await loaders({ directory, expressApp: app })
```

Comment:
self-review: looks like there is some indent issue on my display, someone see the same?

Reply:
I see the same, but I think it's a GitHub UI bug.

Reply:
We have a mix of tab (4) and space (2) indentation.
![Screenshot 2023-04-13 at 9 01 48 AM](https://user-images.githubusercontent.com/116003638/231766898-776a6d07-e4de-470e-a04b-bf6abe6f6100.png)

Reply:
Oh 😦 how did that happened, i ll fix it 💪

Reply:
should be fixed now :)